TrafficCapture/dockerSolution/src/main/docker/elasticsearchWithSearchGuard/Dockerfile

```diff
@@ -5,8 +5,6 @@ RUN echo y | /usr/share/elasticsearch/bin/elasticsearch-plugin install https://m
 RUN pushd /usr/share/elasticsearch/plugins/search-guard-7/tools ; chmod ugo+x ./install_demo_configuration.sh ; yes | ./install_demo_configuration.sh ; popd
 ENV ELASTIC_SEARCH_CONFIG_FILE=/usr/share/elasticsearch/config/elasticsearch.yml
 ENV PROXY_TLS_CONFIG_FILE=/usr/share/elasticsearch/config/proxy_tls.yml
-# without this line, elasticsearch will complain that there aren't enough nodes
-RUN echo "discovery.type: single-node" >> $ELASTIC_SEARCH_CONFIG_FILE
```

Comment:
Removed this due to not noticing this behavior in single node clusters, and it was causing issues in the multiple nodes situation.

Reply:
Just updated the docker compose file to set this again (when in single node build mode) instead of having it built in the image.